src/V3Active.cpp

```diff
@@ -648,5 +648,5 @@ class ActiveVisitor final : public VNVisitor {
 void V3Active::activeAll(AstNetlist* nodep) {
     UINFO(2, __FUNCTION__ << ": " << endl);
     { ActiveVisitor{nodep}; }  // Destruct before checking
-    V3Global::dumpCheckGlobalTree("active", 0, dumpTreeLevel() >= 3);
+    V3Global::dumpCheckGlobalTree("active", 0, dumpTreeEitherLevel() >= 3);
```

Comment:
Thanks for renaming this.  Can you please make a new pull request just with this rename across all of the appropriate functions, and the function declared to match dumpTreeLevel()?  This will greatly shrink #4715 patch size.  Thanks.

(Note one way to make this is just do the "git diff" to a file and manually edit to just have the patches that create this function.)